README.md

```diff
@@ -1,143 +1,415 @@
+
+Folks in Orbit 0 have the dual responsibility to engage both inside the community and advocate for it outside to all of its various stakeholders.
+
+# Using the levels
+
+This section covers how to set up orbit levels and get the most out of them.
+
+## Choosing levels
+
+Love and reach are values that you measure, but **orbit level is a choice that you make for each member based on where you believe they can best serve (and will be best served by) the community**.
+
+> Jamie is creating content and coming to each of our meetups, plus she's volunteered to speak next month. Her activity level is consistent with our Fans, so let's put her into Orbit 2.
+
+Generally speaking, the higher the love—measured as the maximum contribution they've made—should correlate with the orbit level. High love members should go into orbits 1 and 2, with lower love members being further out at 3 and 4. Here's an example of you can map love to orbit level:
+
+| Orbit level (possible) | Love (from) | Love (to) |
+| ---------------------- | ----------- | --------- |
+| 4 - Observers          | 0           | 2         |
+| 3 - Users              | 3           | 6         |
+| 2 - Fans               | 7           | 8         |
+| 1 - Ambassadors        | 9           | 10        |
+
+Love is the maximum score for a recent completed activity.
+
+The example table can serve as a guide, but remember that orbit level is ultimately a choice that should factor in your intuition about where in the community the member will best be served.
+
+## Level distribution
+
+How many members should go in each level? What distribution should we be aiming for in our community? These questions will affect how you choose orbit levels, so it's good to have an idea of what your community's distribution should look like before you do that.
+
+No community is the same, but here's a reasonable starting point for many communities. Roughly speaking, each level is 10 times as large as the previous one.
+
+| Orbit Level | Size | Percentage |
+| ----------- | ---- | ---------- |
+| Ambassadors | 10   | ~1%        |
+| Fans        | 100  | ~10%       |
+| Users       | 1000 | ~90%       |
+| Observers   | ∞    |            |
+
+Observers play a mostly passive, audience-style role in the community, so the size of the level is not bounded.
+j
+There are a few motivations for having targets around the distribution of orbit levels. First, members need to see visible examples of other members doing work at the next level and build relationships with them, before they consider taking the next step. That can’t happen if there are too few Ambassadors relative to Fans or too few Fans relative to Users. The second motivation is to make sure the community is getting all the different contribution types that it needs. For example, when an Ambassador posts a question on the forum, Fans can help start the conversation, Users can continue it, and Observers can read it.
```

Comment:
This section seems like the least clear-cut of them all. I'm not sure I fully grok the first point. The second point seems a lot more practical, and I like the example. 

Also, it seems like there's a missing point about the time commitment from Orbit 0 required to deliver a solid experience to each orbit level.

Reply:
Tried to make it clearer and added a tip about Orbit 0 commitment